sdk/schedules/schedule.ipynb

```diff
@@ -0,0 +1,368 @@
+    "from datetime import datetime\n",
+    "from dateutil import tz\n",
+    "from azure.ai.ml.constants import TimeZone, RecurrencePattern\n",
+    "\n",
+    "schedule_name=\"simple_sdk_create_schedule_recurrence\"\n",
+    "\n",
+    "schedule_start_time = datetime.now(tz=tz.tzutc())\n",
+    "recurrence_trigger = ScheduleRecurrenceTrigger(\n",
+    "    frequency=\"day\",\n",
+    "    interval=1,\n",
+    "    schedule=RecurrencePattern(hours=10, minutes=[0, 1]),\n",
+    "    start_time=schedule_start_time,\n",
+    "    time_zone=TimeZone.UTC,\n",
+    "    create_job=pipeline_job\n",
+    ")\n",
+    "\n",
+    "job_schedule = JobSchedule(\n",
+    "    name=schedule_name,\n",
+    "    trigger=recurrence_trigger,\n",
+    "    create_job=pipeline_job\n",
+    ")\n",
+    "\n",
+    "\n",
+    "# ml_client.schedules.begin_create_or_update(schedule=job_schedule)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.1.2 Create schedule with with cron expression"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "create_schedule_cron"
+   },
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import JobSchedule,ScheduleCronTrigger,ScheduleRecurrenceTrigger # put it to upper once SDK work is ready\n",
+    "\n",
+    "schedule_name=\"simple_sdk_create_schedule_cron\"\n",
+    "\n",
+    "cron_trigger = ScheduleCronTrigger(\n",
+    "    expression=\"15 10 * * *\",\n",
+    "    start_time=\"2022-07-10T10:00:00\", # start time \n",
+    "    time_zone= \"Eastern Standard Time\" # time zone of expression\n",
+    ")\n",
+    "\n",
+    "job_schedule = JobSchedule(\n",
+    "    name=schedule_name,\n",
+    "    trigger=cron_trigger,\n",
+    "    create_job=pipeline_job\n",
+    ")\n",
+    "\n",
+    "\n",
+    "ml_client.schedules.begin_create_or_update(schedule=job_schedule)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.2 Disable the schedule"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "disable_schedule"
+   },
+   "outputs": [],
+   "source": [
+    "ml_client.schedules.begin_disable(name=schedule_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2.3 Check the detail of the schedule"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "show_schedule"
+   },
+   "outputs": [],
+   "source": [
+    "ml_client.schedules.get(name=schedule_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2.4 List schedules in a workspace"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "list_schedule"
+   },
+   "outputs": [],
+   "source": [
+    "ml_client.schedules.list()"
```

Comment:
May do something like below to have better output 

schedules = ml_client.schedules.list()
[s.name for s in schedules]

Reply:
updated